src/pm-conversations/pmConversationsSelectors.js

```diff
@@ -58,7 +53,7 @@ export const getRecentConversations: Selector<PmConversationData[]> = createSele
         /* $FlowFixMe: The keys of unreadPms are logically numbers, but because it's an object they
          end up converted to strings, so this access with string keys works.  We should probably use
          a Map for this and similar maps. */
-        unreadPms[recipient.ids] || unreadHuddles[recipient.ids],
+        unreadPms[recipient.unreadsKey] || unreadHuddles[recipient.unreadsKey],
```

Comment:
Hmm, I'm seeing that `recipient.unreadsKey` ends up being `string | any`, not sure why. Then again, `recipient.ids` before this commit is also `string | any`. 🤷

Reply:
Yeah, that seems to be some kind of bug in the types Flow shows for the IDE.

If you try using the value by e.g. taking some property that strings don't have, it'll show exactly the error you'd hope it would.